ext/php7/logging.h

```diff
@@ -5,12 +5,8 @@
 #include "configuration.h"
 
 inline void ddtrace_log_err(const char *message) {
-#if PHP_VERSION_ID < 80000
     TSRMLS_FETCH();
     php_log_err((char *)message TSRMLS_CC);
```

Comment:
In another followup PR, we could remove the `TSRMLS_*` references for PHP 7 and PHP 8. Then this would become: 

```suggestion
    php_log_err((char *)message);
```

Reply:
I am leaving this for a separate PR as I want to discuss this further.